stdlib/src/memory/anypointer.mojo

```diff
@@ -91,7 +91,7 @@ struct AnyPointer[T: Movable](
             An AnyPointer which contains the address of the argument.
         """
         return __mlir_op.`pop.pointer.bitcast`[_type = Self.pointer_type](
-            __get_lvalue_as_address(arg)
+            Reference(arg)
```

Comment:
**Suggestion** I think this should be `Reference(self).get_unsafe_pointer()` to be explicit.